src/app/view/components/form-email/form-email.component.html

```diff
@@ -3,9 +3,9 @@
   <input type="email" matInput formControlName="email" placeholder="Ex. mariorossi@gmail.com" />
   <mat-icon matSuffix>email</mat-icon>
   <mat-error *ngIf="emailForm.controls['email'].hasError('email') && !emailForm.controls['email'].hasError('required')">
-    Inserisci un indirizzo email valido
+    Please enter a valid email address
   </mat-error>
   <mat-error *ngIf="emailForm.controls['email'].hasError('required')">
-    L'email è <strong>obbligatoria</strong>
+    This field is <strong>required</strong>
   </mat-error>
```

Comment:
magari vedete se si riesce a fare un "microcomponente" che "eredita" da mat-error chiamato tipo required-field-error?

Reply:
il problema è che mat-error deve fare riferimento al form che genera l'errore e imho non credo sia worth it farlo passare per un componente figlio

Reply:
capito. segno resolved